addons/sys_intercom/XEH_postInit.sqf

```diff
@@ -3,14 +3,46 @@
 // Exit if ACE3 not loaded
 if (!isClass (configFile >> "CfgPatches" >> "ace_interact_menu")) exitWith {};
 
-["Tank", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
-["Car_F", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
-["Air", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
-["Boat_F", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Tank", "init", FUNC(initVehicleIntercom), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Car_F", "init", FUNC(initVehicleIntercom), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Air", "init", FUNC(initVehicleIntercom), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Boat_F", "init", FUNC(initVehicleIntercom), nil, nil, true] call CBA_fnc_addClassEventHandler;
 
 if (!hasInterface) exitWith {};
 
-ADDPFH(DFUNC(vehicleCrewPFH), 1.1, []);
+["vehicle",
+    {
```

Comment:
Usually this is done so:
```sqf
["vehicle", {
    // code
}] call CBA_fnc_addPlayerEventHandler;
```

Reply:
There is a lot of code like this in ACRE2.... maybe we do a cleanup PR?

Reply:
Definitely should. But this is new code, so let's do it proper. :)